usaspending_api/download/filestreaming/s3_handler.py

```diff
@@ -9,44 +8,35 @@
 
 class S3Handler:
     """
-    This class acts a wrapper for S3 URL Signing
+    This class acts as a wrapper for S3 URL Signing
     """
 
-    def __init__(self, bucket_name, redirect_dir, region=settings.USASPENDING_AWS_REGION,
-                 environment=settings.DOWNLOAD_ENV):
+    def __init__(self, bucket_name, redirect_dir, region=None, environment=None):
```

Comment:
This feels nitpicky, but leaving the defaults here lets the caller know what the defaults are.  Changing them to `None` makes me think there are no defaults but we find out down below that there actually are defaults.

Now I'm wondering how many times I've done this.....

Reply:
I'm open to having a discussion on this. I prefer not to use global variables as defaults and would rather have the caller provide the values. I viewed this change as a step in the right direction, not the final iteration of code.